src/Components/NavBar/NavBar.jsx

```diff
@@ -84,25 +144,24 @@ export class NavBar extends Component {
     const { searchText } = this.state;
     return (
       <React.Fragment>
-        <Navbar fixed="top" collapseOnSelect expand="lg" variant="dark">
-          <Navbar.Brand>
+        <Navbar fixed="top" className="flex-nowrap flex-row" expand="lg" variant="dark">
+          <Navbar.Brand className="d-none d-md-block">
             <Logo />
           </Navbar.Brand>
-          <Navbar.Toggle aria-controls="responsive-navbar-nav" />
-          <Navbar.Collapse id="responsive-navbar-nav">
+          {this.renderHamburgerIcon(faBars)}
+          {this.displaySearchBox ? (
             <Nav className="mr-auto search-container">
```

Comment:
I noticed that the select option on search was not working did you try to fix that

Reply:
Yes @dusmel I am fixing it now